Rakefile

```diff
@@ -313,6 +323,7 @@ def get_all_project_dir_names()
     Dir.entries(File.dirname(__FILE__)).each do |entry|
         next if not Dir.exist?(entry)
         next if not entry.start_with?("project_")
+        next if entry.end_with?("efs")
```

Comment:
Why exclude this project?

Reply:
Ugh I forgot to take that out. (Was temporarily excluding it because it's not yet committed to the master branch.)